src/app/InceptumApp.ts

```diff
@@ -28,5 +30,13 @@ export class InceptumApp extends BaseApp {
       this.logger.debug('Postgres Detected - Adding Plugin');
       this.register(new PostgresPlugin());
     }
+    if (this.hasConfig('sqsClient')) {
```

Comment:
These keys are case sensitive, and in the plugin classes they are written starting with a capital "S". You need to pick one, or the plugin will not work.